book/src/user/lightwalletd.md

```diff
@@ -0,0 +1,248 @@
+# Running lightwalletd with zebra
+
+Starting on [v1.0.0-beta.10](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.10), the Zebra RPC methods are fully featured to run a lightwalletd service backed by zebrad.
+
+Contents:
+
+- [Download and build Zebra](#download-and-build-zebra)
+- [Configure zebra for lightwalletd](#configure-zebra-for-lightwalletd)
+  - [RPC section](#rpc-section)
+  - [State section](#state-section)
+  - [Network section](#network-section)
+- [Sync Zebra](#sync-zebra)
+- [Download and build lightwalletd](#download-and-build-lightwalletd)
+- [Sync lightwalled](#sync-lightwalled)
+- [Run tests](#run-tests)
+- [Connect wallet to lightwalletd](#connect-wallet-to-lightwalletd)
+  - [Download and build the cli-wallet](#download-and-build-the-cli-wallet)
+  - [Run the wallet](#run-the-wallet)
+- [Lightwalletd and nginx](#lightwalletd-and-nginx)
+
+## Download and build Zebra
+[#download-and-build-zebra]: #download-and-build-zebra
+
+```console
+git clone https://github.com/ZcashFoundation/zebra
+cd zebra
+cargo build --release
+```
+
+Zebra binary will be at `target/release/zebrad`.
+
+## Configure zebra for lightwalletd
+[#configure-zebra-for-lightwalletd]: #configure-zebra-for-lightwalletd
+
+We need a zebra configuration file. First, we create a file with the default settings:
+
+```console
+./target/release/zebrad generate -o zebrad.toml
```

Comment:
Let's put it in the default location for Linux, so users don't have to remember it on the command-line.

We should also link to the default locations for other OSes:
https://docs.rs/dirs/latest/dirs/fn.preference_dir.html

We could also change `zebrad generate` so it automatically puts the file in this location.

```suggestion
zebrad generate -o ~/.config/zebrad.toml
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/4526/commits/2d6b1e746700a06f9c56827b2f367ab493ea5f69